diff --git a/Gruntfile.js b/Gruntfile.js
index 5d41d71..bba35a5 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -114,6 +114,7 @@ module.exports = function( grunt ) {
 					banner: "/*! jQuery v<%= pkg.version %> | " +
 						"(c) 2005, 2013 jQuery Foundation, Inc. | " +
 						"jquery.org/license\n" +
+						"\n Modified by awef \n" +
 						"//@ sourceMappingURL=jquery.min.map\n" +
 						"*/"
 				}
