diff --git a/src/event.js b/src/event.js
index 621464f..0ef5dee 100644
--- a/src/event.js
+++ b/src/event.js
@@ -405,8 +405,7 @@ jQuery.event = {
 		}
 
 		// Determine handlers that should run if there are delegated events
-		// Avoid non-left-click bubbling in Firefox (#3861)
-		if ( delegateCount && !(event.button && event.type === "click") ) {
+		if ( delegateCount ) {
 
 			// Pregenerate a single jQuery object for reuse with .is()
 			jqcur = jQuery(this);
