diff --git a/jquery.mockjax.js b/jquery.mockjax.js
index 5c595f2..b5470df 100644
--- a/jquery.mockjax.js
+++ b/jquery.mockjax.js
@@ -255,7 +255,9 @@
 			getAllResponseHeaders: function() {
 				var headers = '';
 				$.each(mockHandler.headers, function(k, v) {
-					headers += k + ': ' + v + "\n";
+					if ( typeof v === "string" ) {
+						headers += k + ': ' + v + "\n";
+					}
 				});
 				return headers;
 			}
