diff --git build/post-compile.js build/post-compile.js
index 98b8b84..55aa457 100644
--- build/post-compile.js
+++ build/post-compile.js
@@ -6,7 +6,7 @@ var print = require( "sys" ).print,
 	version = fs.readFileSync( "version.txt", "utf8" ),
 	// License Template
 	license = "/*! jQuery v@VERSION http://jquery.com/ | http://jquery.org/license */";
-
+	license += "\n/* このファイルは、オリジナルのjQueryをread.crx用にawefがカスタマイズした物です。 */"
 
 // Previously done in sed but reimplemented here due to portability issues
 src = src.replace( /^(\s*\*\/)(.+)/m, "$1\n$2" ) + ";";
diff --git src/effects.js src/effects.js
index 8009171..741dace 100644
--- src/effects.js
+++ src/effects.js
@@ -569,6 +569,7 @@ function defaultDisplay( nodeName ) {
 			// No iframe to use yet, so create it
 			if ( !iframe ) {
 				iframe = document.createElement( "iframe" );
+				iframe.src = "/view/empty.html";
 				iframe.frameBorder = iframe.width = iframe.height = 0;
 			}
 
