diff --git src/event.js src/event.js
index 5094683..383a0ed 100644
--- src/event.js
+++ src/event.js
@@ -417,8 +417,8 @@ jQuery.event = {
 		event.delegateTarget = this;
 
 		// Determine handlers that should run if there are delegated events
-		// Avoid disabled elements in IE (#6911) and non-left-click bubbling in Firefox (#3861)
-		if ( delegateCount && !event.target.disabled && !(event.button && event.type === "click") ) {
+		// Avoid disabled elements in IE (#6911)
+		if ( delegateCount && !event.target.disabled ) {
 
 			for ( cur = event.target; cur != this; cur = cur.parentNode || this ) {
 				selMatch = {};
