diff --git a/build/post-compile.js b/build/post-compile.js
index afcebf2..a407e57 100644
--- a/build/post-compile.js
+++ b/build/post-compile.js
@@ -6,7 +6,7 @@ var print = require( "sys" ).print,
 	version = fs.readFileSync( "version.txt", "utf8" ),
 	// License Template
 	license = "/*! jQuery v@VERSION jquery.com | jquery.org/license */";
-
+	license += "\n/* このファイルは、オリジナルのjQueryをread.crx用にawefがカスタマイズした物です。 */";
 
 // Previously done in sed but reimplemented here due to portability issues
 src = src.replace( /^(\s*\*\/)(.+)/m, "$1\n$2" ) + ";";
