diff --git a/src/event.js b/src/event.js
index 97a4088..3479c94 100644
--- a/src/event.js
+++ b/src/event.js
@@ -402,8 +402,7 @@ jQuery.event = {
 
 		// Find delegate handlers
 		// Black-hole SVG <use> instance trees (#13180)
-		// Avoid non-left-click bubbling in Firefox (#3861)
-		if ( delegateCount && cur.nodeType && (!event.button || event.type !== "click") ) {
+		if ( delegateCount && cur.nodeType ) {
 
 			for ( ; cur !== this; cur = cur.parentNode || this ) {
 
