diff --git a/src/event.js b/src/event.js
index 902b2b5..4dedfdb 100644
--- a/src/event.js
+++ b/src/event.js
@@ -379,8 +379,7 @@ jQuery.event = {
 		}
 
 		// Determine handlers that should run if there are delegated events
-		// Avoid non-left-click bubbling in Firefox (#3861)
-		if ( delegateCount && !(event.button && event.type === "click") ) {
+		if ( delegateCount ) {
 
 			for ( cur = event.target; cur != this; cur = cur.parentNode || this ) {
 
